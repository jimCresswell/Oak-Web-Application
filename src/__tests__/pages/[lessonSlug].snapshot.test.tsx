import { render } from "@testing-library/react";

import LessonPage from "../../pages/lessons/[lessonSlug]";

it("renders homepage unchanged", () => {
  const { container } = render(<LessonPage />);
  expect(container).toMatchSnapshot();
});
