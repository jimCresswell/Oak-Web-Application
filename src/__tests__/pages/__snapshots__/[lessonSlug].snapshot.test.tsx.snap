// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="title"
    >
      Oak
    </div>
    <form
      class="search"
    >
      <input
        aria-label="Search"
        placeholder="Search"
        type="search"
        value=""
      />
      <button
        type="submit"
      >
        Submit
      </button>
      <div>
        <label>
           Key Stage 
          foundation
          <input
            id="custom-checkbox-foundation"
            name="foundation"
            type="checkbox"
            value="foundation"
          />
        </label>
      </div>
      <div>
        <label>
           Key Stage 
          1
          <input
            id="custom-checkbox-1"
            name="1"
            type="checkbox"
            value="1"
          />
        </label>
      </div>
      <div>
        <label>
           Key Stage 
          2
          <input
            id="custom-checkbox-2"
            name="2"
            type="checkbox"
            value="2"
          />
        </label>
      </div>
      <div>
        <label>
           Key Stage 
          3
          <input
            id="custom-checkbox-3"
            name="3"
            type="checkbox"
            value="3"
          />
        </label>
      </div>
      <div>
        <label>
           Key Stage 
          4
          <input
            id="custom-checkbox-4"
            name="4"
            type="checkbox"
            value="4"
          />
        </label>
      </div>
    </form>
    <div
      class="signInButtonWrapper"
    >
      <a
        href="/sign-in"
      >
        Sign in
      </a>
    </div>
  </header>
  <main
    class="main"
  >
    <nav
      aria-label="Breadcrumb"
      class="nav"
    >
      <ol
        class="ol"
      >
        <li
          class="li"
        >
          <a
            href="/"
          >
            [key-stage]
          </a>
        </li>
        <li
          class="li"
        >
          <span
            class="outer-wrapper"
            style="height: 20px; width: 20px;"
          >
            <span
              class="inner-wrapper"
              style="height: 20px; width: 20px;"
            >
              <svg
                fill="none"
                height="100%"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2.5"
                viewBox="0 0 24 24"
                width="100%"
                xmlns="http://www.w3.org/2000/svg"
              >
                <polyline
                  points="9 18 15 12 9 6"
                />
              </svg>
            </span>
          </span>
          <a
            href="/"
          >
            Subjects
          </a>
        </li>
        <li
          class="li"
        >
          <span
            class="outer-wrapper"
            style="height: 20px; width: 20px;"
          >
            <span
              class="inner-wrapper"
              style="height: 20px; width: 20px;"
            >
              <svg
                fill="none"
                height="100%"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2.5"
                viewBox="0 0 24 24"
                width="100%"
                xmlns="http://www.w3.org/2000/svg"
              >
                <polyline
                  points="9 18 15 12 9 6"
                />
              </svg>
            </span>
          </span>
          <a
            href="/"
          >
            [subject-name]
          </a>
        </li>
        <li
          class="li"
        >
          <span
            class="outer-wrapper"
            style="height: 20px; width: 20px;"
          >
            <span
              class="inner-wrapper"
              style="height: 20px; width: 20px;"
            >
              <svg
                fill="none"
                height="100%"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2.5"
                viewBox="0 0 24 24"
                width="100%"
                xmlns="http://www.w3.org/2000/svg"
              >
                <polyline
                  points="9 18 15 12 9 6"
                />
              </svg>
            </span>
          </span>
          <a
            href="/"
          >
            [unit-name]
          </a>
        </li>
      </ol>
    </nav>
    <header
      class="header"
    >
      <h1>
        Test lesson
      </h1>
      <div>
        <a
          class="button primary large leadingIcon"
          href="/"
        >
          <span
            class="iconWrapper"
          >
            <span
              class="outer-wrapper"
              style="height: 16px; width: 16px;"
            >
              <span
                class="inner-wrapper"
                style="height: 16px; width: 16px;"
              >
                <svg
                  fill="none"
                  height="100%"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2.5"
                  viewBox="0 0 24 24"
                  width="100%"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4"
                  />
                  <polyline
                    points="7 10 12 15 17 10"
                  />
                  <line
                    x1="12"
                    x2="12"
                    y1="15"
                    y2="3"
                  />
                </svg>
              </span>
            </span>
          </span>
          <span
            class="labelSpan"
          >
            Download
          </span>
        </a>
        <a
          class="button primary large leadingIcon"
          href="/"
        >
          <span
            class="iconWrapper"
          >
            <span
              class="outer-wrapper"
              style="height: 16px; width: 16px;"
            >
              <span
                class="inner-wrapper"
                style="height: 16px; width: 16px;"
              >
                <svg
                  fill="none"
                  height="100%"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2.5"
                  viewBox="0 0 24 24"
                  width="100%"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <circle
                    cx="18"
                    cy="5"
                    r="3"
                  />
                  <circle
                    cx="6"
                    cy="12"
                    r="3"
                  />
                  <circle
                    cx="18"
                    cy="19"
                    r="3"
                  />
                  <line
                    x1="8.59"
                    x2="15.42"
                    y1="13.51"
                    y2="17.49"
                  />
                  <line
                    x1="15.41"
                    x2="8.59"
                    y1="6.51"
                    y2="10.49"
                  />
                </svg>
              </span>
            </span>
          </span>
          <span
            class="labelSpan"
          >
            Share
          </span>
        </a>
        <a
          href="/"
        >
          <span
            class="outer-wrapper"
            style="height: 24px; width: 24px;"
          >
            <span
              class="inner-wrapper"
              style="height: 24px; width: 24px;"
            >
              <svg
                fill="currentColor"
                height="24px"
                viewBox="0 0 24 24"
                width="24px"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
                <path
                  d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
        </a>
      </div>
    </header>
    <div
      class="primary-buttons"
    />
  </main>
  <footer
    class="footer"
  >
    © Oak National Academy 
    2022
  </footer>
</div>
`;
